dali/operators/color/brightness_contrast.h

```diff
@@ -60,8 +60,8 @@ class BrightnessContrastOp : public Operator<Backend> {
 
 
   void AcquireArguments(const ArgumentWorkspace &ws) {
-    OperatorBase::AcquireTensorArgument(brightness_, brightness_contrast::kBrightness, ws);
-    OperatorBase::AcquireTensorArgument(contrast_, brightness_contrast::kContrast, ws);
+    this->GetPerSampleArgument(brightness_, brightness_contrast::kBrightness, ws);
```

Comment:
What is the advantage of make this a member function?

Reply:
It's always been one. I've only changed the way of resolving the dependent name problem.